endpoints/getting-started/clients/google-jwt-client.py

```diff
@@ -69,7 +69,6 @@ def make_jwt_request(signed_jwt, url='https://your-endpoint.com'):
     }
     response = requests.get(url, headers=headers)
 
-    response.raise_for_status()
```

Comment:
@DanSanche Does `response.text` need to be returned or is it alright to do the `print()` inside the function?

The `print()` could be moved into the function. 

```python
print(response.text)
response.raise_for_status()
# [END endpoints_jwt_request]
return response.text
```

Reply:
I'm forgetting some of the context around this sample, but I think printing should be fine. That code block looks good to me

Reply:
@TAOXUY would you mind making that change?